.github/workflows/python-lint.yml

```diff
@@ -0,0 +1,21 @@
+name: pre-commit
+
+on: pull_request
+
+jobs:
+  stylecheck:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: install deps
+        run: pip install flake8==3.9.0 black==20.8b1 pre-commit-hooks==2.4.0
+
+      - name: black
+        run: black --check python/hsfs
+
+      - name: flake8
+        run: flake8 --config python/.flake8 python/hsfs
```

Comment:
you can add also to fail when there is trailing whitespace or missing end of lines at file end
```
      - name: trailing-whitespace-fixer
        run: trailing-whitespace-fixer $(find python/hsfs -type f) || exit 1

      - name: end-of-file-fixer
        run: end-of-file-fixer $(find python/hsfs -type f) || exit 1
```